src/abaqus/BasicGeometry/EdgeArray.py

```diff
@@ -64,8 +64,28 @@ def __init__(self, edges: List[Edge]) -> None:
         """
         ...
 
+    @overload
     @abaqus_method_doc
-    def findAt(self, coordinates: tuple, printWarning: Boolean = True) -> Union[Edge, List[Edge]]:
+    def findAt(
+        self, coordinates: Tuple[float, float, float], printWarning: Boolean = True
+    ) -> Edge:
+        ...
```

Comment:
I think this one can be removed because it is covered by the following overload method